### PR TITLE
PDF - word wrap implementation without bands and indents

### PR DESCRIPTION
**Pull request by novsstation**

### CODE REVIEW

File: js/exporter/jspdf/export_data_grid_2.js [hunk 1]
```diff
@@ -37,7 +38,7 @@ function exportDataGrid(doc, dataGrid, options) {
 
                 const currentRowPdfCells = [];
                 currentRowInfo.cellsInfo.forEach(cellInfo => {
-                    const pdfCell = createPdfCell(cellInfo);
+                    const pdfCell = createPdfCell(cellInfo, wordWrapEnabled);
```
Comment (IgnatovDan) [inline js/exporter/jspdf/export_data_grid_2.js]:
```
const pdfCell = createPdfCell(cellInfo); - converts cellInfo into pdfCell
pdfCell.wordWrapEnabled = wordWrapEnabled; - additional settings to cellInfo converting
```
?

File: testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.wordwrap.tests.js [hunk 2]
```diff
@@ -0,0 +1,1649 @@
...
+                });
+            });
+
+            QUnit.test('1 col - 3 text lines. fontSize default, height auto, wordWrap enabled', function(assert) {
+                const done = assert.async();
+                const doc = createMockPdfDoc();
+
+                const dataGrid = createDataGrid({
+                    wordWrapEnabled: true,
+                    columns: [{ caption: 'very long line very long line very long line' }]
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.wordwrap.tests.js:464]: Add pls new test:

columns: [{ caption: 'very long line\nvery long line very long line' }]

Soft and hard line breaks ;-)

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/exporter/jspdf/export_data_grid_2.js (hunk 1 above)
Comment (novsstation) [reply js/exporter/jspdf/export_data_grid_2.js]: Yep, you are absolutely right. Fixed - https://github.com/DevExpress/DevExtreme/pull/18900/commits/768e5957b2e28cc86dc4cb009e337f5a4b7f0b37

File: testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.wordwrap.tests.js (hunk 2 above)
Comment (novsstation) [reply testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.wordwrap.tests.js:464]: Yep, added new test https://github.com/DevExpress/DevExtreme/pull/18900/commits/768e5957b2e28cc86dc4cb009e337f5a4b7f0b37

And sorry, possible my test structure is not intuitive.
A have also another tests, that cover scenario with multiple lines and breaks
https://github.com/DevExpress/DevExtreme/pull/18900/files#diff-086a136fa5207a7cf5a5316dc2e0d9bf67038d4407a6e59d3a52caccaf088347R1597

**Review by KuznetsovVN — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**